merkle-tree/indexed/src/lib.rs

```diff
@@ -245,7 +259,7 @@ where
             .new_low_element
             .hash::<H>(&nullifier_bundle.new_element.value)?;
         let mut zero_bytes_array = BoundedVec::with_capacity(26);
-        for i in 0..16 {
+        for i in 0..address_merkle_tree_inited.merkle_tree.height {
```

Comment:
Using 16 instead of 26 was done on purpose - to fill the proof only until canopy leaves (the rest of it is going to be overwritten anyway).

```suggestion
        for i in 0..address_merkle_tree_inited.merkle_tree.height - address_merkle_tree_inited.canopy_size {
```